Remove gralloc.minigbm from sepolicy

renchenglei opened the pull request:
For virgl, when we ust gralloc.minigbm and hwcomposer.minigbm,
ui will have some issues. Let's use ia hwc and gralloc.
This change help to remove gralloc.minigbm from sepolicy

sysopenci reviewed (commented):
Autobuild started from pull-request-changes on this PR.

FAILURE: CheckBug Bad comments/Bugs

For more information, see: /absp/builders/celadon-autobuild/builds/1447

renchenglei committed:
Remove gralloc.minigbm from sepolicy

For virgl, when we ust gralloc.minigbm and hwcomposer.minigbm,
ui will have some issues. Let's use ia hwc and gralloc.
This change help to remove gralloc.minigbm from sepolicy
